api/desecapi/models.py

```diff
@@ -43,6 +44,20 @@ def create_superuser(self, email, password):
         return user
 
 
+class Token(rest_framework.authtoken.models.Token):
+    key = models.CharField(_("Key"), max_length=40, db_index=True, unique=True)
+    # relation to user is a ForeignKey, so each user can have more than one token
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, related_name='auth_tokens',
+        on_delete=models.CASCADE, verbose_name=_("User")
+    )
+    name = models.CharField(_("Name"), max_length=64)
+
+    class Meta:
+        abstract = False
+        unique_together = (('user', 'name', 'key'),)
```

Comment:
`key` must also be unique, otherwise it may be unclear which user a key points to